<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditNetworkAzsDialog</class>
 <widget class="QDialog" name="EditNetworkAzsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Наименование:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxIsActive">
     <property name="text">
      <string>Сеть в работе</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabServers">
      <attribute name="title">
       <string>Серевера</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabConn2Azs">
      <attribute name="title">
       <string> IP адреса АЗС</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxConnectFromTerm">
         <property name="text">
          <string>Подключение к АЗС с Termadm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLogo">
      <attribute name="title">
       <string>Логотип</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButton">
         <property name="toolTip">
          <string>Загрузить логотип</string>
         </property>
         <property name="text">
          <string>Загрузить логотип</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/Images/folder_48_icon.png</normaloff>:/Images/folder_48_icon.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLogo">
         <property name="text">
          <string>Логотип не загружен</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
